pages/common/git-sed.md

```diff
@@ -0,0 +1,21 @@
+# git sync
+
+> Replace patterns in git-controlled files using sed.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
+
+`git sed '{{find_text}}' '{{replace_text}}'`
+
+- Replace the specified text and then commit the resulting changes with a standard commit message:
+
+`git sed -c '{{find_text}}' '{{replace_text}}'`
+
+- Replace the specified text passing regex flags to `sed`:
```

Comment:
How about this?
```suggestion
- Replace the specified text, using regular expressions:
```

Reply:
That sounds good to me. Thanks for the suggestion.